blocks/network-item/network-item.js

```diff
@@ -94,78 +99,103 @@ function getText(block) {
   return div;
 }
 
-export default function decorate(block) {
-  const blockCfg = readBlockConfig(block);
-  const image = getImage(block);
-  const text = getText(block);
-  const hasImage = !!blockCfg.image;
-
-  let { title } = blockCfg;
+function createTitle(title) {
+  const titleH3 = document.createElement('h3');
+  titleH3.classList.add('title');
+  titleH3.innerText = title;
+  return titleH3;
+}
 
-  if (Array.isArray(blockCfg.title)) {
-    title = title.join('\n');
+function extractVideoFromContent(content, videoImage) {
```

Comment:
From authoring POV, keeping the video in the content seems a bit confusing. Also the logic may be fragile as it assumes the first link to be that of a video. I think you can add a new property `video` which has a link to the video. So we have 2 properties:
- `video-image` - The poster for the video
- `video` - Has the URL of the video modal.
wdyt?

Reply:
I actually tried implementing it like this the first time, but autoblocking the modal didn't work for some reason
I'll try again and see how it goes

Reply:
Ok it seems it was way easier and quicker than expected, not sure what the issue was the first time I tried this